package.json

```diff
@@ -2,20 +2,21 @@
   "name": "react-motion",
   "version": "0.5.2",
   "description": "A spring that solves your animation problems.",
-  "main": "lib/react-motion.js",
+  "main": "dist/react-motion.cjs.js",
+  "module": "dist/react-motion.esm.js",
```

Comment:
should we use `dist/react-motion.jsm` or does not really make much sense for non-nodejs library?